package/kernel/mt76/Makefile

```diff
@@ -507,6 +507,7 @@ define KernelPackage/mt7986-firmware/install
 		$(PKG_BUILD_DIR)/firmware/mt7986_rom_patch.bin \
 		$(PKG_BUILD_DIR)/firmware/mt7986_eeprom_mt7975_dual.bin \
 		$(PKG_BUILD_DIR)/firmware/mt7986_eeprom_mt7976_dual.bin \
+                $(PKG_BUILD_DIR)/firmware/mt7986_eeprom_mt7976_dbdc.bin \
```

Comment:
use tabs instead of spaces

Reply:
Also this changes may need to spilit to another commit

Reply:
A separate commit as part of this PR or a completely different PR just for this change? I assume it should be called something like "mt76: Enable install of mt7986_eeprom_mt7976_dbdc.bin"?

Reply:
As EEPROM data is stored in `factory` partition on flash, there should not even be a need to load this file with default values:
> ```
> 	mediatek,mtd-eeprom = <&factory 0x0>;
> ```

Reply:
So you suggest do remove it and see what happens? If it's missing this file it will complain during boot?

Reply:
Yes. Essentially the expectation is that  the system won't try to load that file, instead loading the data from EEPROM. Only if that fails, the code falls back to load the file through the firmware interface, see also:
https://github.com/torvalds/linux/blob/v6.1/drivers/net/wireless/mediatek/mt76/mt7915/eeprom.c#L238-L244

Reply:
So as the flash partitions are defined in devices tree and we are also booting from that same flash, it is hard for me to imagine why loading the calibration data from flash would fail, ie. what would be a scenario where the device would boot successfully but then require the EEPROM file?

Reply:
I've tested without the file, no error in dmesg, so this can be omitted indeed.

```suggestion
```